python/lsst/daf/butler/datastore/cache_manager.py

```diff
@@ -1194,3 +1202,9 @@ def known_to_cache(self, ref: DatasetRef, extension: str | None = None) -> bool:
 
     def __str__(self) -> str:
         return f"{type(self).__name__}()"
+
+
+class InvalidCacheFilenameError(Exception):
+    """Raised when attempting to register a file in the cache with a name in
+    the incorrect format.
+    """
```

Comment:
Maybe move it to the top, took me a minute to figure out where the name came from when I read the code above?